netlify.toml

```diff
@@ -0,0 +1,20 @@
+[build]
+  base= "client/"
+  command = "yarn build --environment=production"
+
+# production
+[context.master]
+  environment = { HOST="https://applicant-portal.herokuapp.com/" }
```

Comment:
Sorry, what is this syntax? I don't quite get how it works. Looks cool, though!

Reply:
@allthesignals this is [TOML syntax](https://github.com/toml-lang/toml#inline-table), following the example Netlify has in their "file-based config" documentation.  

I reorganized the file to keep things that are consistent across deployments in the `[build]` section, so it's clear the only difference between deploys is the Heroku API deployment they each use.